packages/near-membrane-base/src/membrane.ts

```diff
@@ -1099,77 +1099,6 @@ export function init(
     // future optimization: hoping that proxies with frozen handlers can be faster
     ObjectFreeze(BoundaryProxyHandler.prototype);
 
-    function linkGlobalThis(foreignGlobalThisPointers: Pointer) {
-        ReflectApply(WeakMapProtoSet, proxyTargetToPointerMap, [
```

Comment:
linkage between global objects are now happening at a higher level when needed, not here.